.github/workflows/benchmarks.yml

```diff
@@ -1,6 +1,8 @@
 name: Benchmarking
 on:
-  pull_request:
+  push:
```

Comment:
> Thanks @mlxd for putting this together! I may suggest to add `pyver` in a separate yml file and re-use it with wheels .yml instead of defining it in all of them. What do you think?

Nice idea Ali, but I assumed we cannot add dependencies between separate yaml on Github actions. Is that behaviour changed?

Reply:
It seems composite actions are not fully there yet for the above purpose. I think this is something to revisit once the Actions platform becomes more supportive of shared efforts.

Reply:
Ah! No, it hasn't changed. :crying_cat_face: